### ISSUE TITLE
"Username" Auto Mod Error

### ISSUE THREAD

**Issue by Eposs10** [discord-jda/JDA#2482]
### General Troubleshooting

- [X] I have checked for similar issues on the Issue-tracker.
- [x] I have updated to the [latest JDA version](https://ci.dv8tion.net/job/JDA/lastSuccessfulBuild/)
- [x] I have checked the branches or the maintainers' PRs for upcoming bug fixes.

### Expected Behaviour

no error message

### Code Example for Reproduction Steps

```java
idk its just a normal bot
```

### Code for JDABuilder or DefaultShardManagerBuilder used

```java
JDABuilder builder = JDABuilder.createDefault(token);

        //###########################################
        //# Bot Status
        //###########################################
        builder.setStatus(OnlineStatus.ONLINE);

        //###########################################
        //# Bot activity
        //###########################################
        builder.setActivity(Activity.playing("/help | watches everyone..."));

        //###########################################
        //# Intents die man nutzt
        //###########################################
        builder.enableIntents(
                GatewayIntent.GUILD_MEMBERS,
                GatewayIntent.GUILD_MESSAGES,
                GatewayIntent.GUILD_PRESENCES,
                GatewayIntent.MESSAGE_CONTENT,
                GatewayIntent.GUILD_VOICE_STATES
        );

        //###########################################
        //# Cache
        //###########################################
        builder.setMemberCachePolicy(MemberCachePolicy.ALL);

        dkJDA = builder.build();

        //###########################################
        //# Register listeners
        //###########################################
        dkJDA.addEventListener(
                new EventListener(),
                new CmdManager(),
                new SlashCmd(),
                new ContextMenus()
        );
```

### Exception or Error

```yesyes
JDA 5.0.0-beta.11_3ebfac1
        Unable to resolve value with key channel_id to type unsigned long: null
        AUTO_MODERATION_ACTION_EXECUTION -> {"rule_id":"1101297010218369114","rule_trigger_type":6,"matched_content":"schubi","user_id":"630379516966993931","matched_keyword":"*schubi*","guild_id":"1066462826035691661","action":{"metadata":{},"type":4},"content":""}
net.dv8tion.jda.api.exceptions.ParsingException: Unable to resolve value with key channel_id to type unsigned long: null
        at net.dv8tion.jda.api.utils.data.DataObject.valueError(DataObject.java:886)
        at net.dv8tion.jda.api.utils.data.DataObject.getUnsignedLong(DataObject.java:514)
        at net.dv8tion.jda.internal.entities.automod.AutoModExecutionImpl.<init>(AutoModExecutionImpl.java:42)
        at net.dv8tion.jda.internal.handle.AutoModExecutionHandler.handleInternally(AutoModExecutionHandler.java:46)
        at net.dv8tion.jda.internal.handle.SocketHandler.handle(SocketHandler.java:39)
        at net.dv8tion.jda.internal.requests.WebSocketClient.onDispatch(WebSocketClient.java:1014)
        at net.dv8tion.jda.internal.requests.WebSocketClient.onEvent(WebSocketClient.java:900)
        at net.dv8tion.jda.internal.requests.WebSocketClient.handleEvent(WebSocketClient.java:878)
        at net.dv8tion.jda.internal.requests.WebSocketClient.onBinaryMessage(WebSocketClient.java:1053)
        at com.neovisionaries.ws.client.ListenerManager.callOnBinaryMessage(ListenerManager.java:385)
        at com.neovisionaries.ws.client.ReadingThread.callOnBinaryMessage(ReadingThread.java:276)
        at com.neovisionaries.ws.client.ReadingThread.handleBinaryFrame(ReadingThread.java:996)
        at com.neovisionaries.ws.client.ReadingThread.handleFrame(ReadingThread.java:755)
        at com.neovisionaries.ws.client.ReadingThread.main(ReadingThread.java:108)
        at com.neovisionaries.ws.client.ReadingThread.runMain(ReadingThread.java:64)
        at com.neovisionaries.ws.client.WebSocketThread.run(WebSocketThread.java:45)
```

**Comment by Eposs10** [comment]
i just disabled the automod intents because i dont need them, but idk why they are enabled by default

**Comment by MinnDevelopment** [comment]
Fixed in [5.0.0-beta.12](https://github.com/discord-jda/JDA/releases/tag/v5.0.0-beta.12)
